### PR TITLE
Overview and Installation documentation

### LINKED ISSUE #15: Add install instructions to README.md
The README needs instructions for new contributors to install WiPPLPy and prepare to contribute. These should be similar to those found on https://docs.plasmapy.org/en/latest/contributing/getting_ready.html.

### PR DESCRIPTION
**Pull request by kuchtact**
I've added documentation for how the code works overall and how to install the code. The documentation is built using sphinx. I've also added an init file for the BRB folder.

### CODE REVIEW

**Comment by namurphy** [comment]
Yay for software documentation! 📚🎉🎂💖🎆🪩🌌🥦📚 Thank you for doing this!

**Review by nchurst — APPROVED** [review]
Looks peachy

File: docs/source/code_overview.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.. _code_overview:
```
Comment (broantognetti) [inline docs/source/code_overview.rst:1]: Nice!

**Review by ciortiz — COMMENTED** [review]
Very nice.

**Review by broantognetti — APPROVED** [review]
Wow! Absolutely Fantastic!

**Pull request merged**